version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "2.3.0"
+version in ThisBuild := "2.4.0"
```

Comment:
doesn't this get's bumped automatically during release?

Reply:
According to the docs i wrote 3 years ago, no :)